regolith/tools.py

```diff
@@ -2102,6 +2105,170 @@ def google_cal_auth_flow():
         token.write(creds.to_json())
     # Save the credentials for the next run
 
+def repo_info_complete(target_repo, rc):
+    """checks if repo information is defined and valid in rc.repos (i.e. that
```

Comment:
this is too long.  First line should be less than 70 characters.  then a blank line, then fill in the details.  Check PEPE8 and/or numpy docstrings for details.